### PR TITLE
take into account --include-easyblocks-from-pr when uploading test reports

### PR DESCRIPTION
**Pull request by migueldiascosta**
closes https://github.com/easybuilders/easybuild-framework/issues/3289

### CODE REVIEW

File: easybuild/tools/testing.py [hunk 1]
```diff
@@ -248,18 +249,21 @@ def upload_test_report_as_gist(test_report, descr=None, fn=None):
...
+def post_pr_test_report(pr_nr, repo, test_report, msg, init_session_state, success):
+    """Post test report in a gist, and submit comment in easyconfigs or easyblocks PR."""
 
     github_user = build_option('github_user')
     pr_target_account = build_option('pr_target_account')
-    pr_target_repo = build_option('pr_target_repo') or GITHUB_EASYCONFIGS_REPO
+    if repo == 'easyconfigs':
+        pr_target_repo = build_option('pr_target_repo') or GITHUB_EASYCONFIGS_REPO
+    else:
+        pr_target_repo = build_option('pr_target_repo') or GITHUB_EASYBLOCKS_REPO
```
Comment (akesandgren) [inline easybuild/tools/testing.py:260]: If we manage to enhance this to framework in the future (test reports that is) then we should protect against that by making this a case-stmt instead (of elif) and error out if repo isn't easyconfigs/easyblocks. That way we'll easily catch this one.

File: easybuild/tools/testing.py [hunk 2]
```diff
@@ -276,18 +280,29 @@ def post_easyconfigs_pr_test_report(pr_nr, test_report, msg, init_session_state,
...
-        "Test report by @%s" % github_user,
-        ('**FAILED**', '**SUCCESS**')[success],
+    comment_lines = ["Test report by @%s" % github_user]
+
+    easyblocks_pr_nr = build_option('include_easyblocks_from_pr')
+    if easyblocks_pr_nr:
+        if repo == 'easyconfigs':
+            comment_lines.append("Using easyblocks from https://github.com/%s/%s/pull/%s" % (
+                pr_target_account, GITHUB_EASYBLOCKS_REPO, easyblocks_pr_nr))
+        else:
```
Comment (akesandgren) [inline easybuild/tools/testing.py:290]: Same here ...

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Thanks a lot @migueldiascosta, this is going to be really useful...

**Pull request merged**